packages/idealo.design/src/ui/pages/BlogPage/DetailView.jsx

```diff
@@ -89,44 +86,41 @@ const BlogDetailView = (props) => {
         githubLink = <a href={blogpost.github}><img alt="" src="https://maxcdn.icons8.com/Share/icon/Logos/github_filled1600.png"/></a>
     }
 
+    const htmlBlogContent = draftToHtml(blogpost.blogpostcontent);
 
+    const HtmlToReactParser = require('html-to-react').Parser;
```

Comment:
Hey! Please don't do 'imports/requires' in the midst of a file. They should be put to the top. And, even more important: Please don't mix up import/export (Ecmascript modules) with require/modules.export 
(CommonJS modules). This introduces issues. Please stick to Ecmascript modules.